wdio.conf.js

```diff
@@ -18,6 +18,7 @@ exports.config = {
                 alias: {
                     util: "rollup-plugin-node-polyfills/polyfills/util",
                     path: "rollup-plugin-node-polyfills/polyfills/path",
+                    "node:path": "rollup-plugin-node-polyfills/polyfills/path",
```

Comment:
to fix the error: https://github.com/eslint/eslint/actions/runs/9343712309/job/25713716963#step:5:47